### PR TITLE
add drop_columns option

### PR DESCRIPTION
**Pull request by nishiba**

### CODE REVIEW

File: gokart/task.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ class TaskOnKart(luigi.Task):
     modification_time_check = luigi.BoolParameter(
         default=False,
         description='If this is true, this task will not run only if all input and output files exist,'
-        ' and all input files are modified before output file are modified.',
+                    ' and all input files are modified before output file are modified.',
```
Comment (hirosassa) [inline gokart/task.py]: [nits] 

`' and all input` should be `'and all input` .
(heading white space is unnecessary)

File: gokart/task.py [hunk 2]
```diff
@@ -112,7 +112,7 @@ def make_target(self, relative_file_path: str, use_unique_id: bool = True, proce
         unique_id = self.make_unique_id() if use_unique_id else None
         return gokart.target.make_target(file_path=file_path, unique_id=unique_id, processor=processor)
 
-    def make_large_data_frame_target(self, relative_file_path: str, use_unique_id: bool = True, max_byte=int(2**26)) -> TargetOnKart:
+    def make_large_data_frame_target(self, relative_file_path: str, use_unique_id: bool = True, max_byte=int(2 ** 26)) -> TargetOnKart:
```
Comment (hirosassa) [inline gokart/task.py]: [nits]

> int(2 ** 26)

is this `int()` needed? It is OK when to show your intent explicitly.

**Comment by hirosassa** [comment]
LGTM

**Comment by vaaaaanquish** [comment]
LGTM.

Please write PR description. It may be a simple :)
It will be useful for those who see it later!

**Comment by Hi-king** [comment]
cool!

**Pull request merged**